apps/server/src/modules/authorization/errors/forbidden.error.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { EntityId, IPermissionContext } from '@shared/domain';
+import { ErrorLogMessage, ILoggable } from '@src/core/logger/interfaces/loggable';
+
+// This and the LoggableForbiddenException are alternative implementations.
+// See LoggableForbiddenException for pros and cons.
+export class ForbiddenError extends HttpException implements ILoggable {
+	constructor(private userId: EntityId, private entityId: EntityId, private context: IPermissionContext) {
+		super('Forbidden', HttpStatus.FORBIDDEN);
+	}
+
+	getLogMessage(): ErrorLogMessage {
+		const message = {
+			type: 'FORBIDDEN_EXCEPTION',
```

Comment:
Is the enum value of HttpStatus.FORBIDDEN the same like the string 'FORBIDDEN_EXCEPTION'?

Reply:
No, the enum value of HttpStatus.FORBIDDEN is 403. 

For this type here, @Metauriel and @SevenWaysDP suggested a custom enum to have a finer grained type, as far as I understood. I think like here in the files-storage: https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/files-storage/error/error-status.enum.ts